mairix.c

```diff
@@ -230,9 +230,10 @@ static void parse_rc_file(char *name)/*{{{*/
       line[len-1] = '\0';
     }
 
-    /* Strip trailing comments. */
-    for (p=line; *p && !strchr("#!;%", *p); p++) ;
-    if (*p) *p = '\0';
+    /* Discard comment lines. */
+    if (strchr("#", line[0])) {
```

Comment:
This seems like it might be a bit strict. How about allowing whitespace before the # that introduces a comment. Maybe not. I can be convinced otherwise.

Also, this usage of strchr looks very odd to me. Although it is correct, I had to read it again to determine that.